backend/api/Models/Concepts/File/FilePropertyModel.cs

```diff
@@ -0,0 +1,45 @@
+namespace Pims.Api.Models.Concepts.File
+{
+    public class FilePropertyModel : BaseModel
```

Comment:
This is the minimal set of fields that all FileProperty entities should have, regardless of the type of the file.